src/js/timeManage.js

```diff
@@ -0,0 +1,152 @@
+import 'bootstrap/dist/css/bootstrap.css';
+
+/////init/////
+var t = TrelloPowerUp.iframe();
+
+var members = t.arg("members").members;
+members.forEach(member => {
+    var optionText = member.fullName;
+    var optionValue = member.fullName;
+    $('#members').append(`<option value="${optionValue}"> 
+                                       ${optionText} 
```

Comment:
This code is creating DOM nodes from HTML strings containing potentially unsanitized data, by assigning to innerHTML, jQuery.html, or through similar means. Aside from being inefficient, this is a major security risk. For more information, see https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Safely_inserting_external_content_into_a_page .  ;-)

Reply:
Added DOMPUrify